pkg/codegen/nodejs/gen_program_expressions.go

```diff
@@ -309,13 +309,6 @@ func enumName(enum *model.EnumType) (string, error) {
 	if !ok {
 		return "", fmt.Errorf("Could not get associated enum")
 	}
-	def, err := e.(*schema.EnumType).PackageReference.Definition()
-	if err != nil {
-		return "", err
-	}
-	if name := def.Language["nodejs"].(NodePackageInfo).PackageName; name != "" {
-		pkg = name
-	}
```

Comment:
@iwahbe is there anything I need to be worried about if I delete these lines?

tl;dr working on a bug in nodejs programgen for enums

Current state:

```typescript
import * as pulumi from "@pulumi/pulumi";
import * as pulumiservice from "@pulumi/pulumiservice";

const my_webhook = new pulumiservice.Webhook("my_webhook", {
    active: true,
    displayName: "kyle-repro",
    filters: [@pulumi/pulumiservice.WebhookFilters.DeploymentSucceeded],
    format: @pulumi/pulumiservice.WebhookFormat.Raw,
    organizationName: "pulumi",
    payloadUrl: "https://example.com",
    projectName: "awstypescript",
    stackName: "dev",
}, {
    protect: true,
});
```

PR state:

```typescript
import * as pulumi from "@pulumi/pulumi";
import * as pulumiservice from "@pulumi/pulumiservice";

const my_webhook = new pulumiservice.Webhook("my_webhook", {
    active: true,
    displayName: "kyle-repro",
    filters: [pulumiservice.WebhookFilters.DeploymentSucceeded],
    format: pulumiservice.WebhookFormat.Raw,
    organizationName: "pulumi",
    payloadUrl: "https://example.com",
    projectName: "awstypescript",
    stackName: "dev",
}, {
    protect: true,
});
```

Reply:
If `pulumiservice` has overriden the nodejs package name to be `@pulumi/bar`, you should generate `bar.WebhookFilters.DeploymentSucceeded` instead of `pulumiservice.WebhookFilters.DeploymentSucceeded`. Those lines accommodate for setting `PackageName` in the enums host schema.

Reply:
had internal discussion around test capturing expectations if a different package alias identifier is used.